api/tacticalrmm/agents/views.py

```diff
@@ -210,9 +210,14 @@ def get(self, request, agent_id):
         if token == "err":
             return notify_error("Invalid mesh token")
 
-        control = f"{core.mesh_site}/?login={token}&gotonode={agent.mesh_node_id}&viewmode=11&hide=31"  # type:ignore
-        terminal = f"{core.mesh_site}/?login={token}&gotonode={agent.mesh_node_id}&viewmode=12&hide=31"  # type:ignore
-        file = f"{core.mesh_site}/?login={token}&gotonode={agent.mesh_node_id}&viewmode=13&hide=31"  # type:ignore
+        if core.mesh_auto_login is True:
```

Comment:
no need for the `is True` it's redundant. Since this is a boolean you can just write `if core.mesh_auto_login:`
(also to compare values, would have to use `==` instead of `is`. `is` will compare the types, not values. so you would use `is` if you wanted to check if it was of type boolean like `if type(core.mesh_auto_login) is bool:` otherwise always use `==`

Reply:
Thanks for the tip.  I do not speak idiomatic Python...  :)  If nothing else, it might point at the right spots to put actual idiomatic code...  :)  I look forward to seeing the changes.